_inc/client/components/apps-card/index.jsx

```diff
@@ -1,104 +0,0 @@
-/**
```

Comment:
I think @MichaelArestad was saying that it may be good to keep it in the main Jetpack dashboard for now:

> We do want to leave it on the Jetpack Dashboard for now.

Reply:
lol, right!